pkg/model/components/context.go

```diff
@@ -125,8 +126,16 @@ func Image(component string, clusterSpec *kops.ClusterSpec) (string, error) {
 		return "gcr.io/google_containers/kubedns-amd64:1.3", nil
 	}
 
+
 	if !IsBaseURL(clusterSpec.KubernetesVersion) {
-		return "gcr.io/google_containers/" + component + ":" + "v" + clusterSpec.KubernetesVersion, nil
+		image := "gcr.io/google_containers/" + component + ":" + "v" + clusterSpec.KubernetesVersion
+
+		assetsBuilder := assets.NewAssetBuilder(clusterSpec.Assets)
```

Comment:
OK, but I think we'll have to use the same AssetBuilder if we want this to actually work... in the kubelet pause container it gets put into the OptionsContext